src/tactic/finish.lean

```diff
@@ -360,7 +393,7 @@ local_context >>= case_some_hyp_aux s cont
   The main tactics.
 -/
 
-meta def safe_core (s : simp_lemmas × list name) (cfg : auto_config) : case_option → tactic unit :=
+meta def safe_core (s : simp_lemmas × list name) (ps : list pexpr) (cfg : auto_config) : case_option → tactic unit :=
```

Comment:
As with `done`, could you add doc strings for this, `clarify`, `safe`, `finish`, and the `i` versions? They don't have to be long, since the interactive versions are the primary interface.